interfaces/dbus/backend.go

```diff
@@ -57,6 +71,13 @@ func (b *Backend) Setup(snapInfo *snap.Info, opts interfaces.ConfinementOptions,
 		return fmt.Errorf("cannot obtain dbus specification for snap %q: %s", snapName, err)
 	}
 
+	// core on classic is special
+	if snapName == "core" && release.OnClassic && !release.ReleaseInfo.ForceDevMode() {
```

Comment:
do you really mean `snapName == "core"` and not `snapInfo.Type == snap.TypeCore`?

Reply:
Hm, hm, a good question. I would say "core" is ok here because only on "core" we know that this parth exists.

Reply:
Why is the ForceDevMode check there? I think this is "are we supporting reexec" not "force devmode"